apps/nextjs/src/app/[locale]/manage/tools/docker/docker-table.tsx

```diff
@@ -152,30 +165,78 @@ export function DockerTable(initialData: RouterOutputs["docker"]["getContainers"
 
 interface ContainerActionBarProps {
   selectedIds: string[];
+  selectedContainers: RouterOutputs["docker"]["getContainers"]["containers"];
 }
 
-const ContainerActionBar = ({ selectedIds }: ContainerActionBarProps) => {
+const ContainerActionBar = ({ selectedContainers, selectedIds }: ContainerActionBarProps) => {
   return (
     <Group gap="xs">
-      <ContainerActionBarButton icon={IconPlayerPlay} color="green" action="start" selectedIds={selectedIds} />
-      <ContainerActionBarButton icon={IconPlayerStop} color="red" action="stop" selectedIds={selectedIds} />
-      <ContainerActionBarButton icon={IconRotateClockwise} color="orange" action="restart" selectedIds={selectedIds} />
-      <ContainerActionBarButton icon={IconTrash} color="red" action="remove" selectedIds={selectedIds} />
+      <ContainerActionBarButton
+        icon={IconPlayerPlay}
+        color="green"
+        action="start"
+        selectedIds={selectedIds}
+        selectedContainers={selectedContainers}
+      />
+      <ContainerActionBarButton
+        icon={IconPlayerStop}
+        color="red"
+        action="stop"
+        selectedIds={selectedIds}
+        selectedContainers={selectedContainers}
+      />
+      <ContainerActionBarButton
+        icon={IconRotateClockwise}
+        color="orange"
+        action="restart"
+        selectedIds={selectedIds}
+        selectedContainers={selectedContainers}
+      />
+      <ContainerActionBarButton
+        icon={IconTrash}
+        color="red"
+        action="remove"
+        selectedIds={selectedIds}
+        selectedContainers={selectedContainers}
+      />
+      <ContainerActionBarButton
+        icon={IconCategoryPlus}
+        color="red"
+        action="addToHomarr"
+        selectedIds={selectedIds}
+        selectedContainers={selectedContainers}
+      />
     </Group>
   );
 };
 
 interface ContainerActionBarButtonProps {
   icon: TablerIcon;
   color: MantineColor;
-  action: "start" | "stop" | "restart" | "remove";
+  action: "start" | "stop" | "restart" | "remove" | "addToHomarr";
   selectedIds: string[];
+  selectedContainers: RouterOutputs["docker"]["getContainers"]["containers"];
 }
 
 const ContainerActionBarButton = (props: ContainerActionBarButtonProps) => {
   const t = useScopedI18n("docker.action");
-  const { mutateAsync, isPending } = clientApi.docker[`${props.action}All`].useMutation();
   const utils = clientApi.useUtils();
+  const { openModal } = useModalAction(AddDockerAppToHomarr);
+
+  if (props.action === "addToHomarr") {
+    const handleClick = () => {
+      openModal({
+        selectedContainers: props.selectedContainers,
+      });
+    };
+    return (
+      <Button leftSection={<props.icon />} color={props.color} onClick={handleClick} variant="light" radius="md">
+        {t(`${props.action}.label`)}
+      </Button>
+    );
+  }
+
+  const { mutateAsync, isPending } = clientApi.docker[`${props.action}All`].useMutation();
```

Comment:
I would prefer to have a seperate component for this button as it is not really a container action and rather a homarr action. Also it does do something completly diffrent than the others